src/components/BorrowerProfile/FieldPartnerDetails.vue

```diff
@@ -62,12 +62,13 @@
 				</dd>
 			</div>
 		</dl>
-		<router-link
+
+		<kv-text-link
+			icon="M4,11V13H16L10.5,18.5L11.92,19.92L19.84,12L11.92,4.08L10.5,5.5L16,11H4Z"
```

Comment:
This works, but should probably be a reference to a material design icon import. E.g.,
```
import { mdiArrowRight } from '@mdi/js';
...
data() {
  mdiArrowRight
}
...
<kv-text-link :icon="mdiArrowRight" ... />
```

Reply:
Will do.